app/Kconfig

```diff
@@ -515,8 +533,19 @@ config ZMK_WPM
     bool "Calculate WPM"
     default n
 
-config SENSOR
+config ZMK_KEYMAP_SENSORS
+    bool "Enable Keymap Sensors support"
     default y
+    depends on DT_HAS_ZMK_KEYMAP_SENSORS_ENABLED
+    select SENSOR
+
+if ZMK_KEYMAP_SENSORS
+
+config ZMK_KEYMAP_SENSORS_DEFAULT_TRIGGERS_PER_ROTATION
```

Comment:
What is the difference between this and `ZMK_ENCODERS_DEFAULT_TRIGGERS_PER_ROTATION`? Do we need two different configs (with different defaults) that seem to be for the same thing?